<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Staff</class>
 <widget class="QWidget" name="Staff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Temperatura</string>
      </attribute>
      <widget class="QDial" name="temp_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDial" name="temp_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>170</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcd_temp_1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcd_temp_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Jasność</string>
      </attribute>
      <widget class="QLCDNumber" name="lcd_bright_1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDial" name="bright_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>170</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDial" name="bright_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcd_bright_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Wilgoć</string>
      </attribute>
      <widget class="QLCDNumber" name="lcdNumber_5">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDial" name="dial_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDial" name="dial_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>170</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>temp_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_temp_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>temp_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_temp_1</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bright_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_bright_1</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bright_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_bright_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
